.github/workflows/deploy.yml

```diff
@@ -0,0 +1,41 @@
+# name: Deploy to Netlify
```

Comment:
Do we have any chores for applying those workflows? Or you decided you do it manually?

Reply:
@nvminhtue removed in [b68ff78](https://github.com/manh-t/react-survey/pull/39/commits/b68ff7877896343f127df50578d91e3d4360f9f1)